service/routes.py

```diff
@@ -86,6 +87,26 @@ def list_inventory_records():
     app.logger.info("Returning %d inventory records", len(results))
     return jsonify(results), status.HTTP_200_OK
 
+@app.route("/inventory-records/<product_id>", methods=["DELETE"])
+def delete_inventory_record(product_id):
+    """Deletes inventory record
+
+    @param: product_id is the id of the record that is to be deleted
+    """
+    app.logger.info("Request to delete inventory record with id: %s", product_id)
+    inventory = Inventory()
+    inventory.deserialize(request.get_json())
+    inventory_record = inventory.find((inventory.product_id, inventory.condition))
+    app.logger.info("inventory_recordinventory_record", inventory_record)
```

Comment:
maybe we can rename the info to -> "For Deleting checking if Inventory Record Exists" or something more informative

Reply:
Thanks for your comment, will update!

Reply:
Added to new commit, resolved!